### DISCUSSION TITLE
NetBox maintainers meeting minutes, 23 May 2024

### DISCUSSION THREAD

**Discussion by jeffgdotorg** [netbox-community/netbox#16276] [Announcements]
*NetBox maintainers meeting minutes, 23 May 2024*

_Attending_: Jeremy Stretch, Arthur Hanson, Jeff Gehlbach, Ben Reed, Dan Sheppard

_[Jeremy]  Please prune your assigned issues in GitHub_
- Remove yourself from anything you're not actively working on and update its status label

_[Jeremy] NetBox docs are being migrated to netboxlabs.com_
- Still built from the same repository with the same mkdocs setup
- Traffic has begun cutting over as DNS caches expire
- A few minor styling issues remain, we're aware and will resolve

_[Jeremy] Review of changes to the [issue triage process](https://github.com/netbox-community/netbox/wiki/Issue-Triage-Workflow#issue-triage-flowchart)_
- Stalebot now also cleans up issues with status "revisions needed"
- 7 days to stale + 7 days to closed
- New `backlog` status
- New `complexity:` tags for feature requests
- PRs now go stale after 15 days (closed after +15 days)

_[Jeff] 4.0.2 release; security vulnerability recap_
- The [4.0.2 release](https://github.com/netbox-community/netbox/releases/tag/v4.0.2) included a fix for a high-severity [security vulnerability](https://github.com/netbox-community/netbox/security/advisories/GHSA-32w2-6gm2-7766).
- The release shipped on 14 May, with the advisory following on 20 May.
- ↳ Jeff learned the hard way that only the reporter of an advisory can edit the report body, delaying publication of the advisory
- NetBox Labs cross-functional team defining a cohesive vulnerability intake and response process, which will be published in `SECURITY.md` of all public GitHub projects including NetBox

_[Jeff] Dan’s [PR #15073](https://github.com/netbox-community/netbox/pull/15073) (Add column specific search field to tables)_
- Jeremy and Dan talked through the remaining concerns and identified a way forward that we think will enable us to merge this PR soon

_[Jeff] Retrospective_
- Consensus: Skip this retrospective and let the triage process updates soak in
- Next retrospective remains on the calendar for 27 June

**Comment by opericgithub** [comment]
Regarding the docs migration to netboxlabs.com I would like to know how can one open the following old link now? 

https://docs.netbox.dev/en/develop/release-notes/version-4.0/

It was the docs for develop branch, specifically the release notes. Same applies to feature branch.

**Reply by jeremystretch** [reply]
Use the [stable link](https://netboxlabs.com/docs/netbox/en/stable/release-notes/version-4.0/) for the release notes. The notes for `develop` aren't updated frequently anyway.

**Comment by mrtn-r** [comment]
>[Jeremy] NetBox docs are being migrated to netboxlabs.com

https://netboxlabs.com/docs/netbox/ is not accessible with VPN-Services :/
![image](https://github.com/netbox-community/netbox/assets/153684711/e7b444d4-a9ad-4baa-9e9a-0aca8eb45ecb)

**Reply by candlerb** [reply]
What is "kinsta.cloud", and how does it relate to Netbox?

**Reply by mrtn-r** [reply]
Apparently it's the wordpress-hoster netboxlabs uses:
![image](https://github.com/netbox-community/netbox/assets/153684711/3f740c17-10c8-457c-8640-2c424eb161bb)
![image](https://github.com/netbox-community/netbox/assets/153684711/5e508cfa-2342-4f97-a35c-5ce2441b36bc)
